src/components/dropzone.tsx

```diff
@@ -1,23 +1,32 @@
+import { verify, isValid } from "@govtechsg/oa-verify";
 import { WrappedDocument } from "@govtechsg/open-attestation";
+
 import React from "react";
 import Dropzone from "react-dropzone";
 
 interface DropZoneProps {
   setRawDocument: (doc: WrappedDocument) => void;
+  setDocumentStatus: (bool: boolean | undefined) => void;
 }
-export const DropZone: React.FunctionComponent<DropZoneProps> = ({ setRawDocument }: DropZoneProps) => {
+export const DropZone: React.FunctionComponent<DropZoneProps> = ({
+  setRawDocument,
+  setDocumentStatus,
+}: DropZoneProps) => {
   const onFileDrop = (files: File[]): void => {
     const reader = new FileReader();
 
     reader.onerror = () => {
       alert(`The file uploaded is not a valid Open Attesation file, error: ${reader.error}`);
     };
 
-    reader.onload = () => {
+    reader.onload = async () => {
       try {
         if (reader.result && typeof reader.result === "string") {
+          setDocumentStatus(undefined);
```

Comment:
Not the dropzone responsibility to do this move it home (dropzone just provide the document)